packages/react-components/__tests__/components/Accordion.test.tsx

```diff
@@ -0,0 +1,113 @@
+import Accordion from "../../src/components/Accordion";
+import React from "react";
+import { render } from "@testing-library/react";
+import { createForm, Provider } from "../utils";
+import { renderComponent, DEFAULT_ERROR_MESSAGE } from "../utils";
+import { jsonString } from "@aemforms/af-core";
+import userEvent from "@testing-library/user-event";
+
+const AccordionWithData = {
+    id: "accordion",
+    ":type": "accordion",
+    visible: true,
+    label: { value: "Accordion" },
+    index: 0,
+    items: [
+        {
+          id: "panelcontainer-541adbc656",
+          fieldType: "panel",
+          name: "item_1",
+          visible: true,
+          enabled: true,
+          items: [
+            {
+              id: "checkboxgroup-5f5468260f",
+              fieldType: "checkbox-group",
+              name: "checkboxgroup1675921070268",
+              visible: true,
+              type: "number[]",
+              required: false,
+              enumNames: [
+                "Item 1",
+                " Item 2"
+              ],
+              label: {
+                value: "Check Box Group",
+                visible: true,
+              },
+              enum: [
+                0,
+                1
+              ],
+            },
+          ],
+          label: {
+            value: "Item 1",
+            visible: true,
+           
+          },
+        },
+        {
+          id: "panelcontainer-e48ec263e1",
+          fieldType: "panel",
+          name: "item_3",
+          visible: true,
+          enabled: true,
+          items: [
+            {
+              id: "numberinput-d5f36deec8",
+              fieldType: "number-input",
+              name: "item_16824325701471682432584410",
+              visible: true,
+              type: "string",
+              label: {
+                value: "Number Input",
+                visible: true,
+              },
+            }
+          ],
+          label: {
+            value: "Number",
+            visible: true,
+          }
+        }
+      ],
+  };
+
+  test("Tab should open on which it is clicked and others should be collapsed", () => {
```

Comment:
first tabs should default open and add test cases

Reply:
add test cases for accessibility

Reply:
Added the logic for the same